site/about/index.html

```diff
@@ -1,14 +1,153 @@
+</head>
+
+<body>
+  <nav><a href="/">Home</a><a href="/about">About</a></nav>
+  <h1>Main</h1>
+  <div class="wrapper">
+    <div class="nav-1">
+      <div class="language">
+        <h1>Language</h1>
+      </div>
+      <div class="connect">
+        <h2>Connect</h2>
+      </div>
+      <div class="give">
+        <h2>Give</h2>
+      </div>
+      <div class="apply">
+        <h2>Apply</h2>
+      </div>
+    </div>
+    <div class="nav-2">
+      <div>
+        <h1>Boston Art</h1>
+      </div>
+      <div>
+        <h1>Academy</h1>
+      </div>
+      <div>
+        <h2>About</h2>
+      </div>
+      <div>
+        <h2>Academics</h2>
+      </div>
+      <div>
+        <h2>Admissions</h2>
+      </div>
+      <div>
+        <h2>Community</h2>
+      </div>
+      <div>
+        <h2>Events & Exhibitions</h2>
+      </div>
+      <div>
+        <h2>Founation</h2>
+      </div>
+    </div>
+    <div class="banner">
+      <h1>About</h1>
+    </div>
+    <div class="nav-3">
+      <div>
+        <h3>Mission</h3>
+      </div>
+      <div>
+        <h3>Our Story</h3>
+      </div>
+      <div>
+        <h3>New Campus</h3>
+      </div>
+      <div>
+        <h3>Our Graduates</h3>
+      </div>
+    </div>
+    <div class="our-mission">
+      <div class="mission">
+        <h4>Our Mission</h4>
+        <div>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer nibh nunc, viverra eu euismod ultricies,
+            vehicula a lectus. Maecenas dolor sem, consequat in odio in, aliquam hendrerit dui. Donec quis laoreet
+            massa,
+            vitae blandit ex. Suspendisse ut risus mi. Etiam at venenatis libero. In eget tincidunt nisl. Fusce
+            sagittis
+            felis ut orci dictum vehicula. Donec efficitur nisl eu quam ultrices luctus. Lorem ipsum dolor sit amet,
+            consectetur adipiscing elit. Nullam in condimentum erat. Mauris rhoncus euismod erat vitae semper. Fusce
+            placerat tempor tempus. Aenean fringilla pharetra orci quis commodo.</p>
+        </div>
+        <div>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer nibh nunc, viverra eu euismod ultricies,
+            vehicula a lectus. Maecenas dolor sem, consequat in odio in, aliquam hendrerit dui. Donec quis laoreet
+            massa,
+            vitae blandit ex. Suspendisse ut risus mi. Etiam at venenatis libero. In eget tincidunt nisl. Fusce
+            sagittis
+            felis ut orci dictum vehicula. Donec efficitur nisl eu quam ultrices luctus. Lorem ipsum dolor sit amet,
+            consectetur adipiscing elit. Nullam in condimentum erat. Mauris rhoncus euismod erat vitae semper. Fusce
+            placerat tempor tempus. Aenean fringilla pharetra orci quis commodo.</p>
+        </div>
+      </div>
+    </div>
+    <div class="our-story">
+      <h5> Our Story</h5>
+      <div class="history">
+        <div class="date-1">
```

Comment:
Same as my other note. Try just giving these a class of `date` and style them all through 1 class instead of 4 different ones.